tests/acceptance/config/behat.yml

```diff
@@ -122,7 +122,7 @@ default:
         - '%paths.base%/../features/apiGraph'
       context: *common_ldap_suite_context
       contexts:
-        - RoleAssignmentContext:
+        - SettingsContext:
```

Comment:
don't we use `RoleAssignmentContext`?

Reply:
I just renamed it, and put some new methods